cozy/view_model/library_view_model.py

```diff
@@ -164,7 +155,10 @@ def open_library(self):
         self._notify("library_view_mode")
 
     def book_files_exist(self, book: Book) -> bool:
-        return any(os.path.exists(chapter.file) for chapter in book.chapters)
+        for chapter in book.chapters:
+            if os.path.exists(chapter.file):
```

Comment:
Nitpick: [`os.path.exists`](https://docs.python.org/3/library/os.path.html#os.path.exists) will return `True` if path is an existing directory. [`os.path.isfile`](https://docs.python.org/3/library/os.path.html#os.path.isfile) might be considered to limit it to files.
```suggestion
            if os.path.isfile(chapter.file):
```